Dockerfile

```diff
@@ -33,9 +21,36 @@ COPY pkg/ pkg/
 COPY patches/ patches/
 COPY templates/ templates/
 
-# Build
+###### Dev stage - start ######
+
+FROM common AS dev
+
 USER root
 
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o model-registry
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as dev-build
+
+WORKDIR /
+COPY --from=dev /workspace/model-registry .
+USER 65532:65532
+
+ENTRYPOINT ["/model-registry"]
+
+###### Dev stage - end ######
+
+FROM common as builder
```

Comment:
running this locally, `as` on this line and line 32 show a few errors in the console

```
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 32)
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 42)
 ```